tests/ui/invalid/invalid-rustc_legacy_const_generics-issue-123077.rs

```diff
@@ -0,0 +1,28 @@
+//@ only-x86_64
+
+const fn foo<const U: i32>() -> i32 {
+    U
+}
+
+fn main() {
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, || ());
+    //~^ invalid argument to a legacy const generic
+
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, 5 + || ());
+    //~^ invalid argument to a legacy const generic
+
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, foo::<{ 1 + 2 }>());
+    //~^ invalid argument to a legacy const generic
+
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, foo::<3>());
+    //~^ invalid argument to a legacy const generic
+
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, &const {});
+    //~^ invalid argument to a legacy const generic
+
+    std::arch::x86_64::_mm_blend_ps(loop {}, loop {}, {
+        struct F();
+        //~^ invalid argument to a legacy const generic
+        1
+    });
+}
```

Comment:
Can you add an example like:
```rust
// crate A

#![feature(rustc_attrs)]

#[rustc_legacy_const_generics(0, 1)]
fn foo<const N: usize, const M: usize>() {}

// crate B

fn main() {
    foo(12, (|| 10)());
}
```

to test the case where there are multiple `legacy_const_generic_parameter` arguments. In the example above ideally we would suggest removing *all* legacy const generic arguments and replacing them all with explicit const generic arguments

Reply:
Pushed a testcase using `std::arch::x86_64::_mm_inserti_si64`.